### PR TITLE
Revert changes for node18

### PR DESCRIPTION
**Pull request by stefanosgk**

### CODE REVIEW

**Review by dletran — COMMENTED** [review]
should we update also the changelog for this version?

**Comment by SteveFlames** [comment]
> should we update also the changelog for this version?

hmm, to mention that it was reverted so that we don't get confused in >34.0.0?

**Comment by dletran** [comment]
> > should we update also the changelog for this version?
> 
> hmm, to mention that it was reverted so that we don't get confused in >34.0.0?

yeah I mean if we're reverting the changes without releasing a new version then the changelog is not real anymore so we should clarify that IMO

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - 18
+  - 14
```
Comment (fofi) [inline .travis.yml:3]: Why not using 16 also here?

**Review by fofi — CHANGES_REQUESTED** [review]
Lets also change the version and changelog

**Review by dletran — APPROVED** [review]

**Pull request merged**